cli/src/pcluster/cli/commands/cluster.py

```diff
@@ -0,0 +1,321 @@
+    # CLI
+    name = "start"
+    help = "Starts the compute fleet for a cluster that has been stopped."
+    description = help
+
+    def __init__(self, subparsers):
+        super().__init__(subparsers, name=self.name, help=self.help, description=self.description)
+
+    def register_command_args(self, parser: ArgumentParser) -> None:  # noqa: D102
+        parser.add_argument("cluster_name", help="Starts the compute fleet of the cluster name provided here.")
+
+    def execute(self, args: Namespace, extra_args: List[str]) -> None:  # noqa: D102
+        from pcluster.cli_commands.commands import start
+
+        start(args)
+
+
+class StopClusterCommand(CliCommand):
+    """Implement pcluster stop command."""
+
+    # CLI
+    name = "stop"
+    help = "Stops the compute fleet, leaving the head node running."
+    description = help
+
+    def __init__(self, subparsers):
+        super().__init__(subparsers, name=self.name, help=self.help, description=self.description)
+
+    def register_command_args(self, parser: ArgumentParser) -> None:  # noqa: D102
+        parser.add_argument("cluster_name", help="Stops the compute fleet of the cluster name provided here.")
+
+    def execute(self, args: Namespace, extra_args: List[str]) -> None:  # noqa: D102
+        from pcluster.cli_commands.commands import stop
+
+        stop(args)
+
+
+class ClusterStatusCommand(CliCommand):
+    """Implement pcluster status command."""
+
+    # CLI
+    name = "status"
+    help = "Displays the current status of the cluster."
+    description = help
+
+    def __init__(self, subparsers):
+        super().__init__(subparsers, name=self.name, help=self.help, description=self.description, nowait_arg=True)
+
+    def register_command_args(self, parser: ArgumentParser) -> None:  # noqa: D102
+        parser.add_argument("cluster_name", help="Shows the status of the cluster with the name provided here.")
+
+    def execute(self, args: Namespace, extra_args: List[str]) -> None:  # noqa: D102
+        from pcluster.cli_commands.commands import status
+
+        status(args)
+
+
+class ListClustersCommand(CliCommand):
+    """Implement pcluster list command."""
+
+    # CLI
+    name = "list"
+    help = "Displays a list of stacks associated with AWS ParallelCluster."
+    description = help
+    epilog = "This command lists the names of any CloudFormation stacks named parallelcluster-*"
+
+    def __init__(self, subparsers):
+        super().__init__(subparsers, name=self.name, help=self.help, description=self.description, epilog=self.epilog)
+
+    def register_command_args(self, parser: ArgumentParser) -> None:  # noqa: D102
+        parser.add_argument("--color", action="store_true", default=False, help="Display the cluster status in color.")
+
+    def execute(self, args: Namespace, extra_args: List[str]) -> None:  # noqa: D102
+        from pcluster.cli_commands.commands import list_clusters
+
+        list_clusters(args)
+
+
+class InstancesCommand(CliCommand):
```

Comment:
Should we move it in a instance.py module?

Reply:
Yes let me do this in a later patch where I'll try to better group all commands